templates/rundeck-config.erb

```diff
@@ -31,6 +31,9 @@ dataSource {
   password = "<%= @database_config['password'] %>"
   dialect = "<%= @database_config['dialect'] %>"
   <%- end -%>
+  properties {
+    maxActive = "<%= @database_config['maxactive'] %>"
+  }
```

Comment:
As far as I understand `database_config` is a user provided hash.  If no `maxactive` is contained in it it should not gerenate this config:

```suggestion
<% if @database_config['maxactive'] -%>
  properties {
    maxActive = "<%= @database_config['maxactive'] %>"
  }
<% end -%>
```

In case more properties than `maxActive` are available, maye consider allowing to pass a hash as `@database_config['properties']`:

```suggestion
<% if @database_config['properties'] -%>
  properties {
   <%- @database_config['properties'].each |k, v| -%>
    <%= k %> = "<%= v %>"
   <%- end -%>
  }
<% end -%>
```

A data type to enforce valid keys/values would be a plus but is not strictly necessary since the module currently lacks this.